src/diffusers/models/__init__.py

```diff
@@ -16,12 +16,15 @@
 
 
 if is_torch_available():
-    from .attention import Transformer2DModel
+    from .autoencoder_kl import AutoencoderKL
+    from .dual_transformer_2d import DualTransformer2DModel
+    from .modeling_utils import ModelMixin
     from .prior_transformer import PriorTransformer
+    from .transformer_2d import Transformer2DModel
     from .unet_1d import UNet1DModel
     from .unet_2d import UNet2DModel
     from .unet_2d_condition import UNet2DConditionModel
-    from .vae import AutoencoderKL, VQModel
+    from .vq_model import VQModel
 
 if is_flax_available():
     from .unet_2d_condition_flax import FlaxUNet2DConditionModel
```

Comment:
Should we import `FlaxModelMixin` for consistency?

Reply:
To be honest, I did only the absolute minimum for Flax :sweat_smile: to not spend too much time. Would maybe leave it for a future PR?

Reply:
Sounds good!